src/test/java/com/flightstats/hub/util/SecretsFilterTest.java

```diff
@@ -0,0 +1,27 @@
+package com.flightstats.hub.util;
+
+import org.junit.Test;
+
+import java.util.stream.Stream;
+
+import static org.junit.Assert.*;
+
+public class SecretsFilterTest {
+    @Test
+    public void testRedactionFilter_sensitivePatterns_redacted() {
+        SecretsFilter secretsFilter = new SecretsFilter();
```

Comment:
You can make class member variable and instantiate once

Reply:
done